src/components/Wrapper.js

```diff
@@ -34,13 +34,13 @@ export const InfoBox = styled(WrapperRow)`
   margin-bottom: ${normalize(5)}px;
 `;
 
-export const WrapperWithOrientation = ({ children }) => {
+export const WrapperWithOrientation = ({ children, style }) => {
   const { orientation, dimensions } = useContext(OrientationContext);
   const needLandscapeWrapper =
     orientation === 'landscape' || dimensions.width > consts.DIMENSIONS.FULL_SCREEN_MAX_WIDTH;
 
   if (needLandscapeWrapper) {
-    return <WrapperLandscape>{children}</WrapperLandscape>;
+    return <WrapperLandscape style={style}>{children}</WrapperLandscape>;
```

Comment:
in what cases does the `WrapperLandscape` needs additional styling?

Reply:
just when we have to display a single element on the screen that dosen't belong to one of our screens like detailScreen , HtmlScreen etc ..